knime_extension/src/nodes/spatialtool.py

```diff
@@ -730,7 +738,8 @@ def get_default(cls):
         label="Return only geometries of the same geometry type",
         description="""If selected, the node returns only geometries of the same geometry type as the geometry column 
         from the left (top) input table, otherwise it returns all resulting geometries.""",
-        default_value=lambda v: True if v < knext.Version(1, 1, 0) else False,
+        # default_value=lambda v: True if v < knext.Version(1, 1, 0) else False,
```

Comment:
Why did you change this? This should stay the same to be backward compatible otherwise older workflows will behave differently after the update.

Reply:
I would like to set False  always as the default, which is compatible with the older workflow

Reply:
True the parameter was introduced with 1.1.0 with false as default